app/web_framework/cluster_base_handler.py

```diff
@@ -21,9 +22,29 @@ class ClusterBaseHandler(tornado.web.RequestHandler):
         ItemNotFoundError: http.client.NOT_FOUND,
     }
 
-    def __init__(self, *args, **kwargs):
-        self._logger = log.get_logger(__name__)
-        super().__init__(*args, **kwargs)
+    def _handle_request_exception(self, ex):
```

Comment:
I plan to make another PR to move this logic to either write_error or send_error.